<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <layout class="QVBoxLayout">
   <item><widget class="QLabel"><property name="text"><string>Вход врача</string></property></widget></item>

   <item><widget class="QLineEdit" name="idEdit">
     <property name="placeholderText"><string>ID врача (A1234567)</string></property>
   </widget></item>

   <item><widget class="QLineEdit" name="mailEdit">
     <property name="placeholderText"><string>Email</string></property>
   </widget></item>

   <item><widget class="QLineEdit" name="passEdit">
     <property name="echoMode"><enum>QLineEdit.Password</enum></property>
     <property name="placeholderText"><string>Пароль</string></property>
   </widget></item>

   <item><widget class="QPushButton" name="loginBtn">
     <property name="text"><string>Войти</string></property>
   </widget></item>

   <item><widget class="QLabel" name="statusLabel"/></item>

  </layout>
 </widget>
</ui>
